lld/test/ELF/aarch64-reloc-abs16-addend.test

```diff
@@ -0,0 +1,38 @@
+# REQUIRES: aarch64
+# RUN: yaml2obj %s -o %t.o
+# RUN: ld.lld %t.o -o %t.out
+# RUN: llvm-objdump -t %t.out | FileCheck %s
+
+# CHECK:      SYMBOL TABLE:
+# CHECK-NEXT: 0000000000210120 g     F bar	0000000000000000 _start
+---
+!ELF
+FileHeader:
+  Class: ELFCLASS64
+  Data: ELFDATA2LSB
+  Type: ET_REL
+  Machine: EM_AARCH64
+Sections:
+  - Name: bar
+    Type: SHT_PROGBITS
+    Flags: [SHF_ALLOC, SHF_EXECINSTR]
+  - Name: .debug_frame
+    Type: SHT_PROGBITS
```

Comment:
This creates an empty `.debug_frame`. We technically can report an error, but we don't bother doing it.

It's best to ensure that the size can accommodate the relocation. If `bar` is unrelated to the test, please remove it.